src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -633,13 +574,13 @@ def _create_access_list(signatures, var_accesses):
 
             :param signatures: the list of Signatures for which to create \
                 entries in the list.
-            :type signatures: list of :py:class:`psyclone.core.Signature`
+            :type signatures: List[:py:class:`psyclone.core.Signature`].
             :param var_accesses: object holding details on all variable \
                 accesses in the region to which the data-access clause applies.
-            :type var_accesses: :py:class:`psyclone.core.VariablesAccessInfo`
+            :type var_accesses: :py:class:`psyclone.core.VariablesAccessInfo`.
```

Comment:
No full stop at the end of type specifications please. (See the mark-up rules in the Dev Guide).

Reply:
Weird, I was quite sure I saw some full stops, thus why I added them.